code/modules/clothing/suits/miscellaneous.dm

```diff
@@ -382,6 +382,41 @@
 	icon_open = "varsity_brown_open"
 	icon_closed = "varsity_brown"
 
+/obj/item/clothing/suit/storage/toggle/varsity/med
+	name = "medical varsity"
+	desc = "A varsity jacket in blue, red, and white, featuring an image of the Caduceus on the back, symbolizing not only the art of healing but also your fraternity (or sorority)"
```

Comment:
You can drop the redundant parentheses, leaving just "...your fraternity or sorority."

There should be a comma before "but".

Feel free to use this simplified version which keeps the wording pretty much the same.

> A varsity jacket in blue, red, and white, featuring an image of the Caduceus on the back, symbolizing both the art of healing and your fraternity or sorority.

Reply:
Got it with parentheses. 
Punctuation in English is somewhat mystery for me, but this looks like my blunder
I will use your version